src/Symfony/Component/Security/Http/Session/SessionAuthenticationStrategy.php

```diff
@@ -48,7 +48,7 @@ public function onAuthentication(Request $request, TokenInterface $token)
 
             case self::MIGRATE:
                 // Note: this logic is duplicated in several authentication listeners
-                // until Symfony 5.0 due to a security fix with BC compat
+                // until Symfony 5.0 due to a security fix with BC compat XXX
```

Comment:
What does this mean for 5.0 @weaverryan? What should be done now?

Reply:
removing the duplicated logic in other listener I think

Reply:
this is the only remaining part that I didn't update in this PR
@weaverryan maybe, up for a PR?